srcCpp/PerftestTransport.cxx

```diff
@@ -1052,4 +1098,26 @@ bool PerftestTransport::parseTransportOptions(int argc, char *argv[])
     return true;
 }
 
+bool PerftestTransport::allowsMulticast()
```

Comment:
After discussing with @alfonso-soto 
This function should not check the variable `useMulticast`
```
bool PerftestTransport::allowsMulticast()
{
    return (transportConfig.kind != TRANSPORT_TCPv4
            && transportConfig.kind != TRANSPORT_TLSv4
            && transportConfig.kind != TRANSPORT_WANv4
            && transportConfig.kind != TRANSPORT_SHMEM);
}
```

Reply:
Done.